src/com/linkedin/parseq/Engine.java

```diff
@@ -204,6 +205,14 @@ public boolean awaitTermination(final int time, final TimeUnit unit) throws Inte
     return _terminated.await(time, unit);
   }
 
+  public boolean addPlanActivityListener(final PlanActivityListener planActivityListener) {
+    return _planActivityListeners.add(planActivityListener);
+  }
+
```

Comment:
do we have a use case for this, adding listeners after construction of engine? If not, we can remove these two methods.